typing/cmi_format.ml

```diff
@@ -20,8 +20,7 @@ type pers_flags =
   | Unsafe_string
 
 type error =
-    Not_an_interface of string
-  | Wrong_version_interface of string * string
+    Not_an_interface of string * string option
```

Comment:
P.S.: I plan to reconsider my position above somewhat, but right now I have zero time to work on this further.